packages/aws-cdk/lib/api/util/cloudformation/stack-activity-monitor.ts

```diff
@@ -83,7 +96,7 @@ export class StackActivityMonitor {
   /**
    * Determines which events not to display
    */
-  private startTime = Date.now();
+  private readonly startTime: number;
```

Comment:
Any reason not to leave this as a `Date`?

Reply:
Just simpler implementation, and since it doesn't affect the public API I prefer this.